### PR TITLE
feat(icon): adding new search filled and keyword icons

### PR DESCRIPTION
**Pull request by panvourtsis**
## Description

This PR is adding two new icons. 
- SearchFilled
- Keyword

https://www.figma.com/design/YX5qwIRXYippaBdyoWHTGx/%F0%9F%8E%89-Ictinus-V5.0-[Current]?node-id=26036-682&t=2h06aqy6yjvHe2cQ-0

## Screenshot

Screenshot 2024-06-13 at 3 46 43 PM

### CODE REVIEW

File: src/components/Icon/assets/generic/keyword.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (kostasdano) [inline src/components/Icon/assets/generic/keyword.svg:1]: remove width and height attributes as they alter the final size of the icon

File: src/components/Icon/assets/generic/search filled.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (kostasdano) [inline src/components/Icon/assets/generic/search filled.svg:1]: same here

Comment (panvourtsis) [reply src/components/Icon/assets/generic/search filled.svg]: resolved at https://github.com/Orfium/orfium-ictinus/pull/823/commits/a3879a6f77774c1d9d1257360457ec6bebbb5f79

File: src/components/Icon/assets/generic/keyword.svg (hunk 1 above)
Comment (panvourtsis) [reply src/components/Icon/assets/generic/keyword.svg]: resolved at https://github.com/Orfium/orfium-ictinus/pull/823/commits/a3879a6f77774c1d9d1257360457ec6bebbb5f79

File: src/components/Icon/assets/generic/search filled.svg (hunk 2 above)
Comment (kostasdano) [reply src/components/Icon/assets/generic/search filled.svg]: @panvourtsis dont forget to update snapshots because tests are broken atm

Comment (panvourtsis) [reply src/components/Icon/assets/generic/search filled.svg]: fixed at https://github.com/Orfium/orfium-ictinus/pull/823/commits/9b6079d2978d744e61fe9d1282113fec22ecc4b6

**Review by kostasdano — APPROVED** [review]

**Pull request merged**